tests/sentry/api/endpoints/test_organization_member_team_details.py

```diff
@@ -205,8 +192,7 @@ def org(self):
 
     def test_member_must_request_access_to_join_team(self):
         self.login_as(self.member.user)
-        resp = self.get_response(self.org.slug, self.member.id, self.team.slug)
-        assert resp.status_code == 202
+        self.get_success_response(self.org.slug, self.member.id, self.team.slug)
```

Comment:
should we still look specifically for a 202 here? (and a 201 above)

Reply:
Yes this is the only file where we make assertions about the API so we should be explicit about status code if that's something we want to be strict about.